rasa/nlu/constants.py

```diff
@@ -39,19 +39,28 @@
 
 NUMBER_OF_SUB_TOKENS = "number_of_sub_tokens"
 
-MESSAGE_ATTRIBUTES = [TEXT, INTENT, RESPONSE]
-DENSE_FEATURIZABLE_ATTRIBUTES = [TEXT, RESPONSE]
+MESSAGE_ATTRIBUTES = [TEXT, INTENT, RESPONSE, ACTION_NAME, ACTION_TEXT]
+# the dense featurizable attributes are essentially text attributes
+DENSE_FEATURIZABLE_ATTRIBUTES = [TEXT, RESPONSE, ACTION_TEXT]
+NAME_ATTRIBUTES = [INTENT, ACTION_NAME]
```

Comment:
this system seem to be working good, the name of the constant should be different maybe